azure-pipelines.yml

```diff
@@ -6,7 +6,7 @@
 variables:
   tmpDir: 'D:\Temp'
 

```

Comment:
Can you explain the new strategy in the PR's description, mostly for future reference?